### PR TITLE
added authors in DESCRIPTION

### PR DESCRIPTION
**Pull request by clsu22**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
 Version: 0.0.0.9000
-Authors@R: 
-    person(given = "Haoyu",
-           family = "Su",
-           role = c("aut", "cre"),
-           email = "haoyu.su666@gmail.com",
-           comment = c(ORCID = "YOUR-ORCID-ID"))
+Authors@R: c(person("Zhengyang (Zoe) Pan", "Developer", role = c("aut", "cre"),
+                      email = "haoyu.su.ubc@gmail.com"),
+              person("Haoyu (Clara) Su", "Developer", role = "aut"),
```
Comment (zoepan00) [inline DESCRIPTION]:
```suggestion
Authors@R: c(person("Zhengyang (Zoe) Pan", "Developer", role = c("aut", "cre")),
              person("Haoyu (Clara) Su", "Developer", role = "aut",
                      email = "haoyu.su.ubc@gmail.com"),
```

File: DESCRIPTION [hunk 2]
```diff
@@ -1,12 +1,11 @@
 Package: rimager
 Title: Process images
 Version: 0.0.0.9000
```
Comment (zoepan00) [inline DESCRIPTION]:
```suggestion
Version: 1.0.0
```

**Review by zoepan00 — CHANGES_REQUESTED** [review]
Please have a look of suggesions

**Comment by clsu22** [comment]
something wrong, closed for now

**Pull request closed without merging**